001-project-file-structure/src/index.html

```diff
@@ -5,6 +5,7 @@
     <title>Angular Ex1</title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" type="text/css" link="https://v4-alpha.getbootstrap.com/dist/css/bootstrap.min.css">
```

Comment:
This is not right to hardcode css or script libraries in the html, after the application is compiled they will be exposed to the public view, you should install those libraries via bower or npm, following the implementation